federation/apis/core/v1/defaults.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
+	v1.RegisterDefaults(scheme)
```

Comment:
Yes, if you include upstream objects you must also include their defaulters.  Once you do this though you no longer need the next line.